apps/studio/components/interfaces/Settings/Integrations/VercelIntegration/VercelSection.tsx

```diff
@@ -65,7 +66,39 @@ const VercelSection = ({ isProjectScoped }: { isProjectScoped: boolean }) => {
 
   // We're only supporting one Vercel integration per org for now
   // this will need to be updated when we support multiple integrations
-  const vercelIntegration = vercelIntegrations?.[0]
+  const vercelIntegration = vercelIntegrations?.[0] || {
+    id: '123',
+    inserted_at: new Date().toString(),
+    updated_at: new Date().toString(),
+    added_by: {
+      username: 'a',
+      id: 'a',
+      primary_email: 'as',
+    },
+    organization: { slug: '123' },
+    integration: { name: 'Vercel' },
+    connections: [
+      {
+        id: '123',
+        inserted_at: new Date().toString(),
+        updated_at: new Date().toString(),
+        env_sync_targets: ['production', 'preview'],
+        added_by: {
+          username: 'a',
+          id: 'a',
+          primary_email: 'as',
+        },
+        supabase_project_ref: '123',
+        foreign_project_id: '123',
+        organization_integration_id: '123',
+        metadata: {
+          id: '123',
+          name: 'asd',
+          framework: 'nextjs' as VercelFramework,
+        },
+      },
+    ],
+  }
```

Comment:
Is the dummy data still needed here? If yes, would you mind adding an explainer comment?